contracts/evmscript/EVMScriptRunner.sol

```diff
@@ -36,7 +36,7 @@ contract EVMScriptRunner is AppStorage, EVMScriptRegistryConstants {
     /**
     * @dev copies and returns last's call data. Needs to ABI decode first
     */
-    function returnedDataDecoded() internal view returns (bytes ret) {
+    function returnedDataDecoded() internal pure returns (bytes ret) {
```

Comment:
I'm not 100% sure, but I think this should be a `view` function since it's relying on manipulating memory. Correct me if I'm wrong though; my understanding was that `pure` is for functions that only use their params.

Reply:
Note to self: clearly I was wrong. `view` is only necessary for `SLOAD` and `SSTORE`.